src/python/grpcio/grpc/beta/implementations.py

```diff
@@ -105,10 +105,33 @@ def metadata_call_credentials(metadata_plugin, name=None):
     A CallCredentials object for use in a GRPCCallOptions object.
   """
   if name is None:
-    name = metadata_plugin.__name__
+    name = metadata_plugin.__class__.__name__
   return CallCredentials(
       _low.call_credentials_metadata_plugin(metadata_plugin, name))
 
+def google_call_credentials(credentials):
```

Comment:
I think the naming "google_call_credentials" and "access_token_call_credential" is good, let's get a second opinion from @nathanielmanistaatgoogle  or @soltanmm.  If everything else looks good, I think it's fine to submit this change and revisit the naming if necessary.